forem/forem
# Remove redirect on comments#update

## What type of PR is this? (check all applicable)

## Description
This refactors `comments#update` to avoid doing a redirect causing a race condition when busting cache.

## Related Tickets & Documents
https://github.com/forem/forem/issues/10338

## QA Instructions, Screenshots, Recordings

1. Fire up the app locally and sign in.
2. Leave a comment on an article.
3. Update the comment you just made and make sure it doesn't error out. It should be the same UX as before just without a redirect - it goes back to the comment index view and displays the updated comment content.
4. Leave a comment on a podcast.
5. Repeat step 3.
6. Add yourself to an organization in your local DB and publish an article under an organization.
7. Leave a comment on the article.
8. Repeat step 3.

## Added tests?

## Added to documentation?


![dog_racing_human_in_push_car](https://media.giphy.com/media/hqh8NpKgBMRTa/giphy.gif)
